src/Main.java

```diff
@@ -1,7 +1,76 @@
-// Press Shift twice to open the Search Everywhere dialog and type `show whitespaces`,
-// then press Enter. You can now see whitespace characters in your code.
+
 public class Main {
     public static void main(String[] args) {
+        System.out.println("Задача 1");
+
+        var clientOs = 1;
+        if (clientOs == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке.");}
+        else if (clientOs == 0) {
+            System.out.println("Установите версию приложения для IOS по ссылке.");}
+
+        System.out.println();
+        System.out.println("Задача 2");
+
+        var clientOsTwo = 1;
+        var clientDeviceYear = 2015;
+        if ((clientOsTwo == 1) && (clientDeviceYear >= 2015))  {
```

Comment:
```suggestion
        if (clientOsTwo == 1 && clientDeviceYear >= 2015)  {
```
скобки не нужны дополнительные (ниже то же самое)